### PR TITLE
chore: some minor cleanup

### PR DESCRIPTION
**Pull request by ariostas**
A little bit of cleanup before making the first official deployment. Fixed some metadata and added a few curricula to match what was on the old training center.

### CODE REVIEW

File: data/curricula.yaml [hunk 1]
```diff
@@ -58,3 +58,16 @@
...
+        - id: julia
+        - id: unroot
+    - name: "Miscellaneous"
+      description: "Other useful tools and trainings"
+      modules:
+        - id: simpleanalysis
+    - name: "Planned, in early development, or archived"
+      description: "Useful modules that are not yet ready for prime time"
+      modules:
+        - id: uproot
```
Comment (klieret) [inline data/curricula.yaml:73]: I think @jpivarski mentioned that `uproot` has been superseded by the Scikit-HEP tutorial (and not sure what the maintenance state of `uproot` is)

File: data/curricula.yaml [hunk 2]
```diff
@@ -58,3 +58,16 @@
...
         - id: lhcb-analysis-essentials
+    - name: "Julia corner"
+      description: "Learn Julia for fast and easy code!"
+      modules:
+        - id: julia
+        - id: unroot
+    - name: "Miscellaneous"
+      description: "Other useful tools and trainings"
+      modules:
+        - id: simpleanalysis
```
Comment (klieret) [inline data/curricula.yaml:69]: I think this one is mostly payload for the CI/CD tutorial, not sure if we should list it seperately

**Review by klieret — APPROVED** [review]
I think some of the things copied over might be legacy, but you decide :)

File: data/curricula.yaml (hunk 1 above)
Comment (ariostas) [reply data/curricula.yaml:73]: Ah okay, I'll remove it again.

**Pull request merged**